dotCMS/src/main/webapp/WEB-INF/web.xml

```diff
@@ -505,11 +508,14 @@
 		<servlet-name>DotGraphQLHttpServlet</servlet-name>
 		<url-pattern>/api/v1/graphql</url-pattern>
 	</servlet-mapping>
+
+	<!--
```

Comment:
If you are removing this because it is now dynamically registered with the annotation on DotRestApplication we should clean this up and remove both the serlet and servlet-mapping entries for this and also remove the ReloadableServletContainer class it is referencing.    We have previously removed the need for the ReloadableServletContainer anyway and the web.xml could just as easily have referenced the base org.glassfish.jersey.servlet.ServletContainer.   The only reason this was not converted before was to not modify the web.xml.    My concern we have to be aware of is if we have customers mapping custom web.xml files and how we go about patching or informing the need to update these as they will have problems if their instance still has the old servlet mapping.   We may need to discuss with cloud engineering on the impact of changes to the web.xml like this.

To note: this change can be the first step for us to use annotations for all the rest of the servlets and filters until we could get rid of the web.xml, it is no longer used by default in current servlet spec and is only there for backward compatibility.   Having to map in changes into the xml for customers is a pain point for cloud engineering.

Reply:
I found problems with the two types of instructions that had to coexist. 
Dynamic Rest endpoints wouldn't load until I removed the mapping from the web.xml